src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -173,17 +173,136 @@ private void ViewLoadedHandler(object sender, RoutedEventArgs e)
         private void ViewModel_RequestZoomToFit(BoundingBox bounds)
         {
             var prevcamDir = watch_view.Camera.LookDirection;
-            watch_view.ZoomExtents(bounds.ToRect3D(.05));
-            //if after a zoom the camera is in an undefined position or view direction, reset it.
-            if(watch_view.Camera.Position.ToVector3().IsUndefined() || 
-                watch_view.Camera.LookDirection.ToVector3().IsUndefined() || 
-                watch_view.Camera.LookDirection.Length == 0)
+            if (watch_view.Camera is HelixToolkit.Wpf.SharpDX.PerspectiveCamera perspectiveCam)
```

Comment:
Do we need to account for the case where the camera is Orthographic?

Reply:
hmm, let me look into what happened before.

Reply:
yeah, your're right @aparajit-pratap that would have been supported before, I'll move in the code for ortho cameras as well from helix.

Reply:
![Screenshot 2023-12-12 at 2 40 15 PM](https://github.com/DynamoDS/Dynamo/assets/508936/734f3332-247a-455b-808a-0753d4417719)
![Screenshot 2023-12-12 at 2 40 12 PM](https://github.com/DynamoDS/Dynamo/assets/508936/685ad2f6-641f-4c8a-bb38-8ffb004f4c16)

done.

Reply:
Wait, how do we change the camera from perspective to orthographic? Is there a UI option?

Reply:
it can be done from an extension - theres a property on the helix viewport object